includes/multi-currency/class-currency.php

```diff
@@ -191,13 +214,14 @@ public function set_rounding( $rounding ) {
 	 */
 	public function jsonSerialize() {
 		return [
-			'code'       => $this->code,
```

Comment:
I know this function wasn't added in this PR, but we could also add type hinting here since it should always return an `array`. :)

Reply:
Thanks! Fixed in f288f19 too.